app/controllers/pxe_controller/pxe_image_types.rb

```diff
@@ -191,7 +189,7 @@ def pxe_image_type_get_node_info(treenodeid)
     else
       @right_cell_div = "pxe_image_type_details"
       nodes = treenodeid.split("-")
-      @record = @pxe_image_type = PxeImageType.find_by_id(from_cid(nodes.last))
+      @record = @pxe_image_type = PxeImageType.find(from_cid(nodes.last))
```

Comment:
afair find_by_id will return nil if not found vs find will raise exception if not found?

Reply:
yes, but the code around assumes that the value exists

Reply:
```
@right_cell_text = _("System Image Types \"%{name}\"") % {:name => @pxe_image_type.name}
```

`@pxe_image_type.name`